### PR TITLE
Feat/tak 43 faq page implementation

### PR DESCRIPTION
**Pull request by thunderTheGun**

### CODE REVIEW

File: components/FAQ/molecules/Collage.tsx [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+
+import FaqImage from 'public/assets/gyik.png';
+import {
+  CollageContainer,
+  LargerSquare,
+  SmallerSquare,
+  ImageContainer,
+} from 'components/FAQ/molecules/Collage.styles';
+
+const Collage = () => (
```
Comment (Roverr) [inline components/FAQ/molecules/Collage.tsx:11]: Let me ask you a question, if these components in molecules are not using atoms, what makes them molecules?

**Review by Roverr — COMMENTED** [review]
Good job overall I like what I see!

File: components/FAQ/molecules/Collage.tsx (hunk 1 above)
Comment (thunderTheGun) [reply components/FAQ/molecules/Collage.tsx:11]: This question came up in me as well. I had 2 answers for myself :) 

First, I would argue that the inner implementation contains atoms even if they are not put into standalone files somewhere in a folder that is called 'atoms'.

Second, even if the above is not always the case, sometimes the size and/or complexity of the element just puts them into a bigger category in my head. I mean I could hardly consider them as 'atoms' when atoms are usually a header or a paragraph. This reasoning I would not consider as strong as the first one, I have to admit.

Even though I ended up using them this way, it is still a very thought-provoking question and the starter of an excellent conversation. I personally think that an analogy from chemistry should not induce overthinking or overly strict rules that do not necessarily help development. Meaning that the methodology could be fine in code, even if it breaks the rules and definition of stuff it took inspiration from.

Others, what's your opinion? @Roverr @orosznyet @balintant @mikitovicsmark

Comment (Roverr) [reply components/FAQ/molecules/Collage.tsx:11]: Yeah I can completely agree with this argument 👍

**Review by Roverr — APPROVED** [review]

File: components/FAQ/molecules/Collage.styles.tsx [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+`;
+
+export const ImageContainer = styled.div`
+  width: 272px;
+  height: 160px;
+  position: relative;
+  left: 64px;
+  bottom: 113px;
+
+  @media screen and (min-width: 1200px) {
```
Comment (mikitovicsmark) [inline components/FAQ/molecules/Collage.styles.tsx]: I'd rather go with % values here with a min-width, or min calculation here. Looks weird on tablets IMO.

https://developer.mozilla.org/en-US/docs/Web/CSS/min

File: components/FAQ/molecules/QAndItem.tsx [hunk 3]
```diff
@@ -0,0 +1,69 @@
...
+interface Props {
+  item: {
+    question: string;
+    answer: string;
+  };
+  isLastItem: boolean;
+}
+
+const QAndAItem = ({ item, isLastItem }: Props) => {
+  const [isOpen, setIsOpen] = useState(false);
```
Comment (mikitovicsmark) [inline components/FAQ/molecules/QAndItem.tsx]: I'd highly recommend animating this. Not much effort, and makes the UX a lot better.

I can recommend using this lib, should be pretty lightweight, and easy to use.
https://github.com/Stanko/react-animate-height

File: components/FAQ/molecules/Collage.styles.tsx (hunk 2 above)
Comment (thunderTheGun) [reply components/FAQ/molecules/Collage.styles.tsx]: do you mean it is too small related to the page width? @mikitovicsmark

Comment (thunderTheGun) [reply components/FAQ/molecules/Collage.styles.tsx]: changed this a bit, please review latest push @mikitovicsmark

**Review by mikitovicsmark — APPROVED** [review]

**Pull request merged**